docs/querying/sql-data-types.md

```diff
@@ -92,36 +92,39 @@ the `UNNEST` functionality available in some other SQL dialects. Refer to the do
 > nearly all possible native functionality.
 
 ## Arrays
-Multi-value dimensions may also be converted to standard SQL arrays, either by explicitly converting them with `MV_TO_ARRAY`,
-or implicitly when used within the [array functions](./sql-array-functions.md). `ARRAY` types behave as standard SQL arrays, where
-grouping on them will group on the entire array of values instead of the implicit `UNNEST` that occurs when grouping on
-multi-value dimensions directly or when used with the multi-value functions. Arrays may also be constructed from multiple
-columns using the array functions.
+Druid also has support for `ARRAY` types constructed at query time, though currently lacks the ability to store them in
+segments. `ARRAY` types behave as standard SQL arrays, where results are grouped by matching entire arrays. This is in
+contrast to the implicit `UNNEST` that occurs when grouping on multi-value dimensions directly or when used with the
+multi-value functions. You can convert multi-value dimensions to standard SQL arrays either by explicitly converting
+them with `MV_TO_ARRAY`, or implicitly when used within the [array functions](./sql-array-functions.md). Arrays may
+also be constructed from multiple columns using the array functions.
 
 ## Multi-value strings behavior
 The behavior of Druid [multi-value string dimensions](multi-value-dimensions.md) varies depending on the context of their usage.
 
-When used as `VARCHAR` functions, which are not "aware" that their inputs which claim to be `VARCHAR` might actually have multiple
-values such as `CONCAT`, Druid will map the function across all values in the row. If the row is null or empty, the function will
-recieve `NULL` as its input, otherwise it will be applied to every row value and continue its life as a multi-value VARCHAR.
+When used with standard `VARCHAR` functions which expect a single input value per row, such as `CONCAT`, Druid will map
```

Comment:
```suggestion
When used with standard `VARCHAR` functions, which expect a single input value per row, Druid maps the function across all values in the row. If the row is null or empty, the function receives `NULL` as its input, otherwise it is applied to every row value and continues its life as a multi-value `VARCHAR`.
```
Do you think we can omit the `CONCAT` example in this case? I think the sentence reads better without it.